### ISSUE TITLE
Any suggestion to use TCN for more than 1000 classes?

### ISSUE THREAD

**Issue by mchancan** [philipperemy/keras-tcn#37]

**Comment by philipperemy** [comment]
It's extremely vague. Can you be more precise? I guess you should post it in reddit and not here :)

**Comment by philipperemy** [comment]
One way to deal with it is to consider Hierarchical Softmax.
